pages/reference-react-hooks/usesendtransaction.mdx

```diff
@@ -0,0 +1,130 @@
+import { Tabs } from "nextra/components";
+
+# `useSendTransaction`
+
+`(params?: UseSendTransactionParams): UseSendTransactionReturnType`
+
+## Usage
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['App.tsx', 'main.tsx']}>
+  <Tabs.Tab>
+  ```typescript
+  import { useSendTransaction } from '@safe-global/safe-core-sdk-react-hooks'
+  
+  function App() {
+    const { sendTransaction, isLoading, isError, data, error } = useSendTransaction();
+
+    const handleSendTransaction = () => {
+      sendTransaction({
+        transactions: [
+          {
+            to: '0x...',
+            value: '0x...',
+            data: '0x...',
+          },
+        ],
+      });
+    };
+
+    return (
+      <div>
+        <button onClick={handleSendTransaction} disabled={isLoading}>
+          Send Transaction
+        </button>
+        {isError && <p>Error: {error.message}</p>}
+        {data && <p>Transaction sent: {JSON.stringify(data)}</p>}
+      </div>
+    );
+  }
+  ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+  ```typescript
+  import ReactDOM from 'react-dom/client'
+  import { createConfig, SafeProvider } from '@safe-global/safe-core-sdk-react-hooks'
+  import App from './App.tsx'
+
+  const config = createConfig({
+    chain,
+    provider,
+    signer,
+    safeOptions: {
+      owners,
+      threshold
+    }
+  })
+
+  ReactDOM.createRoot(document.getElementById('root')!).render(
+    <SafeProvider config={config}>
+      <App />
+    </SafeProvider>
+  )
+  ```
+  </Tabs.Tab>
+</Tabs>
+
+{/* <!-- vale on --> */}
+
+## Parameters
+
+- `params` (UseSendTransactionParams, optional): Parameters to customize the hook behavior. It contains:
+  - `config` (SafeConfig, optional): The [`SafeConfig`](./safeconfig.mdx) is used when a `SafeProvider` is not present in the component tree.
```

Comment:
Maybe it would be good to add here that the config must contain a signer property.

Reply:
Same for useConfirmTransaction